packages/backend/discovery/starknet/config.jsonc

```diff
@@ -6,15 +6,31 @@
     "0xae0Ee0A63A2cE6BaeEFFE56e7714FB4EFE48D419",
     "0x283751A21eafBFcD52297820D27C1f1963D9b5b4",
     "0xF6080D9fbEEbcd44D89aFfBFd42F098cbFf92816",
-    "0xbb3400F107804DFB482565FF1Ec8D8aE66747605"
+    "0xbb3400F107804DFB482565FF1Ec8D8aE66747605",
+    "0x0437465dfb5b79726e35f08559b0cbea55bb585c"
   ],
+  "names": {
+    "0x0437465dfb5B79726e35F08559B0cBea55bb585C": "DAI Bridge",
+    "0x283751A21eafBFcD52297820D27C1f1963D9b5b4": "WBTC Bridge",
+    "0xF6080D9fbEEbcd44D89aFfBFd42F098cbFf92816": "USDC Bridge",
+    "0xbb3400F107804DFB482565FF1Ec8D8aE66747605": "USDT Bridge"
+  },
   "overrides": {
+    // Starknet
+    // Make it possible to use the contract name without overriding it
```

Comment:
write `"0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4": "Starknet"` inside names for now

Reply:
I know it's possible. I just don't like it too much now